### DISCUSSION TITLE
Different fields on default booking printable

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#547] [Q&A] [answered]
Hi, I'ld like to change the data that is printed on the booking printable (Afdrukvoorbeeld). Where can I do that?

**Comment by liedekef** [comment, marked as answer]
You can't. The printable booking report is just 1of the provided reports. If you don't like the layout/data: use the CSV report and create your own layout in excel.

**Comment by JanineKall** [comment]
Maybe I was not specific enough. I want the ID field in the CSV and the printable replaced by the Payment ID. 
Would it work if I edited the eme-people.php for that?

**Comment by liedekef** [comment]
In the CSV you can use filters to add/remove fields if something is missing, not in the printable booking report. Changing the code is not recommended and your changes will get removed with each upgrade.
